src/Conference/Plugin.php

```diff
@@ -240,6 +240,7 @@ protected function register_autoloader() {
 		// Load Composer autoload and strauss autoloader.
 		require_once dirname( CONFERENCE_SCHEDULE_FILE ) . '/vendor/vendor-prefixed/autoload.php';
 		require_once dirname( CONFERENCE_SCHEDULE_FILE ) . '/vendor/autoload.php';
+		require_once dirname( CONFERENCE_SCHEDULE_FILE ) . '/vendor/cmb2/cmb2/init.php';
```

Comment:
- I changed to load cmb2 through composer so it is in the vendor directort.
- The file is directly loaded per their documentation. 
- CMB2 changes the main class name with each version update. 
- It also has checks if the class exists.